src/components/DatePicker/DatePicker.stories.js

```diff
@@ -107,9 +105,9 @@ import { SingleDatePicker } from 'react-dates';
   inputIconPosition="after"
   invalidText="A valid value is required"
   labelText="Label datepicker"
-  onBlur={()=>{console.log("Blur")}}
-  onChange={()=>{console.log("onChange")}}
-  onFocus={()=>{console.log("onFocus")}}
+  onBlur={()=>{}}
```

Comment:
In the stories we can have `console.log` since they won't appear in the final build and only in the documentation.